collections/_get-started/installation.html

```diff
@@ -1,33 +1,86 @@
 ---
 title: Installation
 layout: layout-sidenav
-description: A dropdown mechanism that allow users to either show or hide related content.
+description: Installation guide that will help you quicky spin up a site using SGDS
 ---
+{% capture reactjs %}
+npm install @govtechsg/sgds-react
+
+//not required if using cdn
+npm install @govtechsg/sgds-govtech bootstrap-icons
```

Comment:
npm install @govtechsg/sgds bootstrap-icons 

ok i missed this out on react installation guide too

Reply:
@rav3 think u missed this out. Other than that I approved